apps/condo/domains/contact/schema/_internalSyncContactsWithResidentsForOrganizationService.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Generated by `createservice contact._internalSyncContactsWithResidentsForOrganizationService --type mutations`
+ */
+
+const {
+    GQLCustomSchema,
+    getById,
+} = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/contact/access/_internalSyncContactsWithResidentsForOrganizationService')
+const { Contact } = require('@condo/domains/contact/utils/serverSchema')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+
+const _internalSyncContactsWithResidentsForOrganizationService = new GQLCustomSchema('_internalSyncContactsWithResidentsForOrganizationService', {
+    schemaDoc: 'Mutation to sync organization contacts with organization residents. ' +
+        'It takes all organization residents and create contact for each one (if contact with same phone number not exist). ' +
+        'Returns array of created contacts',
+    types: [
+        {
+            access: true,
+            type: 'input _internalSyncContactsWithResidentsForOrganizationInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput! }',
```

Comment:
perhaps it would be better to use `organizationId: ID!`

or add a check that `organization.id` is specified

Reply:
OrganizationWhereUniqueInput have only one field - ID and it is required

Reply:
my mistake :)